### PR TITLE
add ui examples

### PR DESCRIPTION
**Pull request by zhuxinglongbest**
Signed-off-by: zilin.zxl <zilin.zxl@alibaba-inc.com>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 07b88cfadbc1ad79fc1800351d734274cd5739b4

**Comment by skylarCai** [comment]
auto build success,number:235

**Comment by skylarCai** [comment]
autotest fail,number:1740

**Review by qisheng-seu — COMMENTED** [review]
looks ok to me

File: haas_lib_bundles/python/docs/examples/ui_collection/page_connect.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import lvgl as lv
+import wifi_module
+import _thread
+import font_Alibaba_PuHuiTi
+
+RESOURCES_ROOT = "S:/data/pyamp/images/"
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/ui_collection/page_connect.py:7]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**